docs/concepts/sensor-models/sensor-models.md

```diff
@@ -0,0 +1,88 @@
+
+Alone, a CSM is of little practical use. The standard defines an interface but does not define where necessary sensor metadata should be sourced, the format of said metadata, or the implementation of the photogrammetric algorithms used by the sensor. Therefore, a CSM exists within a broader ecosystem. First, some external capability or method must be used to generate image support data (ISD) that can be used to parameterize a CSM. Then, the CSM can parse and use the ISD to extract the parameters (interior and exterior orientation) necessary to perform photogrammetric operations (as defined by the CSM API). Finally, a SET is likely used to orchestrate the pieces of the CSM ecosystem and exploits said pieces in order to use the sensor for some desired operation [(Laura et al., 2020)](https://doi.org/10.1029/2019EA000713)
+
+!!! INFO "CSM -- Quick Definition"
+    The CSM is a standard that defines an API for sensor models, thereby providing a common structure that guarantees interoperability of any sensor models that adhere to the standard.
+
+## The ASC Sensor Model Ecosystem
+The Astrogeology Science Center has created and maintains a suite of software packages that provide end-to-end processing for sensor models, including a metadata specification, CSM-compliant sensor model implementation, and tools to leverage those sensor models for scientific processing.
+
+
+### Abstraction Library for Ephemerides (ALE)
+The [Abstraction Library for Ephemerides](https://github.com/DOI-USGS/ale/) (ALE) provides the tools and information necessary to derive and access a sensor's exterior orientation.  ALE provides a suite of instrument-specific drivers that combine information from a variety of metadata formats (image labels), sensor types, and data sources into a single CSM compliant format. It is important to note that ALE is responsible for abstracting away intermediate reference frames between the body-centered, body-fixed frame and the sensor frame so that the resulting model can transform directly from the sensor frame to the BCBF frame.
+
+ALE uses a collection of drivers and class mixins to provide ISDs for a variety of sensor models.  [Drivers](https://github.com/DOI-USGS/ale/tree/main/ale/drivers) for many major missions are available in ALE.  Each driver is required to provide support for at least one label type (ISIS or PDS3) and one source of SPICE data (ISIS or NAIF). It is important to note that not all drivers currently support all combinations of labels and SPICE data sources.
+
+!!! INFO "ALE -- Quick Definition"
```

Comment:
I like these quick definitions a lot. Like CSM, would this be appropriate at the top of this section on ALE?